src/lib/verifier/prod.ml

```diff
@@ -127,7 +115,8 @@ end
 
 type t = Worker.Connection.t
 
-let create ~conf_dir =
+(* TODO: investigate why conf_dir wasn't being used *)
```

Comment:
The verifier ought to be stateless, not sure why it would need the conf_dir.

Reply:
Yeah I just left it when I went through but it's unused. I'll just remove it.